packages/server-core/src/projects/githubapp/githubapp-helper.ts

```diff
@@ -39,6 +40,7 @@ export const createGitHubApp = () => {
 
 export const getGitHubAppRepos = async () => {
   try {
+    await refreshAppConfig()
```

Comment:
rather than calling this here, we should listen to the `server-setting` feathers service patch calls and update then, rather than lazily when `getGitHubAppRepos` is called